### DISCUSSION TITLE
Windows version. Autostart.

### DISCUSSION THREAD

**Discussion by squromiv** [TechnitiumSoftware/DnsServer#456] [Q&A] [answered]
I need to start technitium service automatically. But every time system reboots, Technitium DNS Service is in "stop" state and I must start it manually from Tray App. Am I doing smth wrong or is it an issue?
Windows Server 2016. System Tray App v.3.0.  Checkbox "Auto Start icon" is set. Service run policy is "Automatic".
![1](https://user-images.githubusercontent.com/15985877/190891241-2e343ee5-bcd1-45f2-a4e8-27bb535cf831.jpg)
![2](https://user-images.githubusercontent.com/15985877/190891244-37fcca1e-b9ad-4bef-a54c-489628b36b54.jpg)
![4](https://user-images.githubusercontent.com/15985877/190891250-006d35b6-0f4a-405a-9738-65af2c9943f8.jpg)

**Comment by ShreyasZare** [comment]
Thanks for the post. Check the DNS server logs from the web panel and see if there are any errors logged. Do share any error that you see here.

**Reply by squromiv** [reply]
> see if there are any errors logged

There is no errors.

**Comment by squromiv** [comment, marked as answer]
UPD
I`ve found out, that problem is not in Technitium. 3-dparty software (OpenServer) stops the service for some reason. Thanks for support.

**Reply by ShreyasZare** [reply]
Thanks for letting me know that the issue is fixed.
